### ISSUE TITLE
Baidu torch model cant download

### ISSUE THREAD

**Issue by MyraBaba** [deepinsight/insightface#1545]
Hi @nttstar 

We cant download from Baidu Torch model (Baidu Yun Pan: e8pw).

Is it possible to put DropBox ?

**Comment by gerald-ftk** [comment]
Agreed, would anyone be able to provide a Google Drive or Dropbox link of the Arcface Pytorch models?

**Comment by nttstar** [comment]
README updated.
https://1drv.ms/u/s!AswpsDO2toNKq0lWY69vN58GR6mw?e=p9Ov5d
